openpype/hosts/houdini/plugins/load/load_vdb.py

```diff
@@ -31,6 +31,7 @@ def load(self, context, name=None, namespace=None, data=None):
 
         # Create a new geo node
         container = obj.createNode("geo", node_name=node_name)
+        is_sequence = bool(context["representation"]["context"].get("frame"))
```

Comment:
Would this be a false negative if the first frame of the sequence is zero (`0`)?

Reply:
It won't as the `frame` is string. But it is place to revisit in v4  (along with so many others)